src/croc/croc.go

```diff
@@ -435,11 +435,17 @@ func (c *Client) Receive() (err error) {
 				)
 				c.ExternalIPConnected = c.Options.RelayAddress
 				usingLocal = true
+				break
```

Comment:
I noticed that this loops over the `discoveries` but never does anything to any of them except the first element, makes sense to abort early then?